src/istio/mixerclient/report_batch.cc

```diff
@@ -67,10 +68,13 @@ void ReportBatch::FlushWithLock() {
   ++total_remote_report_calls_;
   auto request = batch_compressor_->Finish();
   ReportResponse* response = new ReportResponse;
+
+  // TODO(jblatt) should an async call be made while this lock is held?  Can the
+  // request send block()?
   transport_(request, response, [this, response](const Status& status) {
     delete response;
     if (!status.ok()) {
-      GOOGLE_LOG(ERROR) << "Mixer Report failed with: " << status.ToString();
+      MIXER_WARN("Mixer Report failed with: %s", status.ToString().c_str());
```

Comment:
Can we handle timeout logs differently than other failures?  We have a lot of complaints about the logs filling up with these statements... and there isn't anything that can really be done.  I'd like to see some sort of rate limiting or sampling on this to ease the volume -- especially for timeouts.

Reply:
How about a (0 == counter++ % 100) guard?   What frequency would you like?

Reply:
that sounds reasonable to me.  @mandarjog any thoughts here?